src/Plugin/Field/FieldWidget/LocalistUrlWidget.php

```diff
@@ -336,15 +327,15 @@ protected function getGroups($default_value = ''): array {
    * @return array
    *   Form element render array.
    */
-  protected function getPlaces($default_value = ''): array {
+  protected function getPlaces($default_value = null): array {
     $places = $this->fetchLocalistData('places');
     $element = [
       '#type' => 'select',
       '#title' => $this->t('Venues'),
       '#multiple' => FALSE,
       '#options' => [],
       '#empty_option' => '(Optional) Select one:',
```

Comment:
```suggestion
      '#empty_option' => 'Select one:',
```
I thought I had changed that in https://github.com/SU-SWS/stanford_fields/pull/23 but I guess I didn't.  Anyways, that "optional" shouldn't be there, per Ishi.  See also D8CORE-5116.